docker/config.js

```diff
@@ -26,7 +26,11 @@ module.exports = {
 	},
 
 	database: {
-		url: "postgres://miaou:miaou@postgres:5432/miaou",
+    database: "miaou",
```

Comment:
There's some space/tabs issue here, pretty visible in github diff UI :)

Reply:
Ok, I'll fix that